fe/fe-core/src/main/java/org/apache/doris/master/ReportHandler.java

```diff
@@ -481,7 +481,6 @@ private static void sync(Map<Long, TTablet> backendTablets, ListMultimap<Long, L
                             // 1. PUSH finished in BE but failed or not yet report to FE
                             // 2. repair for VERSION_INCOMPLETE finished in BE, but failed or not yet report to FE
                             replica.updateVersionInfo(backendVersion, backendVersionHash, dataSize, rowCount);
-                            replica.setBad(false);
```

Comment:
For reviewer:
In the original design logic, when the status of a replica is bad, it cannot become not bad again.
This is a clerical error and will cause the `admin set replica status` function to fail.